### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by weisJ**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
```
Comment (marko-radosavljevic) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe just leave the numbers, so it's easier to fill?
I think its obvious how steps should look like? Maybe I'm wrong

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
```
Comment (marko-radosavljevic) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: I feel like 'Expected behaviour' and possibly 'Actual behaviour' should be placed before the steps for reproduction

**Review by marko-radosavljevic — COMMENTED** [review]
I think formatting can be better.

Descriptions of each section are not in the new lines, if this is intentional, maybe aligning them would be better.
It also kinda doesn't look like a form, you are not sure if you should delete the descriptions or not, there are no : after each section signalling where the information should be entered.

So maybe each description should be in the new line, italic, and possibly with tags <> to indicate it's expected to be edited? Also : after each bold section header. Or something similar, to make it easier for the eyes to extract all the important info, and to make it more standardized (everyone enters info in the same place).

Other than that, I like it! ^^

**Comment by weisJ** [comment]
> Descriptions of each section are not in the new lines, if this is intentional, maybe aligning them would be better.
>

I think that's an issue with the github rich diff. The descriptions are actually in a new line. This is just a modified version of the default github templates.

> It also kinda doesn't look like a form, you are not sure if you should delete the descriptions or not, there are no : after each section signalling where the information should be entered.
> 
> So maybe each description should be in the new line, italic, and possibly with tags <> to indicate it's expected to be edited? 
>

We could put the descriptions in a comment tag. Then only the person editing can see them.

**Review by marko-radosavljevic — APPROVED** [review]

**Comment by marko-radosavljevic** [comment]
Yup, comments are probably the best solution. Looks good to me

**Pull request merged**